user/mockStoreClient.go

```diff
@@ -12,14 +16,23 @@ func NewMockStoreClient(salt string, returnDifferent, doBad bool) *MockStoreClie
 	return &MockStoreClient{salt: salt, doBad: doBad, returnDifferent: returnDifferent}
 }
 
-func (d MockStoreClient) Close() {}
-
+func (c *MockStoreClient) Close() error {
```

Comment:
even if don't like the d for the client, let's keep it consistent within the file
```suggestion
func (d *MockStoreClient) Close() error {
```